contrib/cirrus/test.sh

```diff
@@ -2,8 +2,9 @@
 
 set -eo pipefail
 
-systemctl enable podman.socket podman.service
-systemctl start podman.socket
+echo "Locate: $(which podman)"
```

Comment:
Minor nit: `which` is horrible and evil, use: `echo "Locate: $(type -P podman)"`